src/main/java/fi/vm/yti/datamodel/api/model/AbstractModel.java

```diff
@@ -186,4 +197,14 @@ public String getContentModified() {
                 LDHelper.curieToProperty("iow:documentation")).toList());
     }
 
+    private void addParentOrganizationId(RDFNode orgRes) {
+        Statement parent = orgRes.asResource().getProperty(
+                ResourceFactory.createProperty(LDHelper.PREFIX_MAP.get("iow") + "parentOrganization"));
+        try {
+            this.editorOrganizations.add(UUID.fromString(parent.getResource().getURI().replaceFirst("urn:uuid:", "")));
+        } catch (Exception ignore) {
```

Comment:
This exception could probably be more specific, if we're always expecting something like `NullPointerException`, and everything else would be unexpected.

Reply:
Changed to ResourceRequiredException